src/new-features-in-517.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default_md
+title: New Features in 5.17 
+title-class: page-title-activemq5
+type: activemq5
+---
+
+[Features](features) > [New Features](new-features) > [New Features in 5.17](new-features-in-517)
+
+
+New Features in 5.17
+----------------------
+
```

Comment:
I would add:
* Spring 5 update (done)
* Log4j2 update (in progress)

Reply:
* JMS 2.0
* JUnit 5 support

Reply:
Let's focus on what the current state is:
* Spring 5 <- will add 
* Log4j2 <- we can add this once it's merged
* JMS 2.0 <- already on
* JUnit 5 <- I haven't seen this done yet? We can always update this once it's merged

* moving the page <- let's do this all at once in a commit so we don't have a weird heterogenous link hierarchy.